ui/app/AppLayouts/Wallet/panels/DAppsWorkflow.qml

```diff
@@ -69,8 +69,8 @@ DappsComboBox {
     signal disconnectRequested(string connectionId)
     signal pairingRequested(string uri)
     signal pairingValidationRequested(string uri)
-    signal connectionAccepted(var pairingId, var chainIds, string selectedAccount)
-    signal connectionDeclined(var pairingId)
+    signal connectionAccepted(string pairingId, var chainIds, string selectedAccount)
```

Comment:
Tried to keep things inline and the pairing ids and topics to be strings in all signals